docs/reference/tab-widgets/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,212 @@
+// tag::cloud[]
+In order to restore the indices and data streams that are missing data:
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
```

Comment:
These pluses are all showing on the page, but I don't think they were intended. Should we removed them?

Reply:
Definitely not intended. I will fix it.